frontend/src/pages/image/_id/index.vue

```diff
@@ -168,7 +168,12 @@ export default defineComponent({
 
     const route = useRoute()
 
-    const image = ref<ImageDetail | null>(singleResultStore.image)
+    const image = ref<ImageDetail | null>(
```

Comment:
This is one part that fixes #3830 (not re-using the store image if the id doesn't match)